ui/pages/confirmation/components/confirmation-network-switch/confirmation-network-switch.js

```diff
@@ -20,82 +18,54 @@ import {
 } from '../../../../../shared/constants/network';
 import { getProviderConfig } from '../../../../ducks/metamask/metamask';
 
-export default function ConfirmationNetworkSwitch({ newNetwork }) {
-  const { chainId, nickname, type } = useSelector(getProviderConfig);
-
+const NetworkIcon = ({ chainId, nickname }) => {
   return (
-    <Box
-      className="confirmation-network-switch"
-      display={Display.Flex}
-      height={BlockSize.Full}
-      justifyContent={JustifyContent.center}
-      marginTop={8}
-    >
-      <Box
-        className="confirmation-network-switch__icon"
-        display={Display.Block}
-      >
+    <Box className="confirmation-network-switch__icon-wrapper">
+      <Box className="confirmation-network-switch__icon">
         {chainId in CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP ? (
-          <SiteIcon
-            icon={CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP[chainId]}
+          <AvatarFavicon
```

Comment:
I know in the issue it says to replace the SiteIcon with AvatarFavicon but because it is to identify a network I would suggest using `AvatarNetwork` here
```suggestion
          <AvatarNetwork
```